hypertrace-gradle-java-convention-plugin/src/main/java/org/hypertrace/gradle/java/convention/JavaConventionExtension.java

```diff
@@ -8,6 +8,7 @@
 
 public class JavaConventionExtension {
   static final String EXTENSION_NAME = "javaConvention";
+  // TODO: resolve if need two different for source & target
```

Comment:
I thought in our usage at least we wouldn't need different. I can add another property if you feel such a case might come up